ietf/meeting/utils.py

```diff
@@ -710,3 +710,34 @@ def handle_upload_file(file, filename, meeting, subdir, request=None, encoding=N
         subprocess.call(['unzip', filename], cwd=path)
 
     return None
+
+def new_doc_for_session(type_id, session):
+    typename = DocTypeName.objects.get(slug=type_id)
+    ota = session.official_timeslotassignment()
+    sess_time = ota and ota.timeslot.time
```

Comment:
We'll need to make this TZ aware - or we could do it now like this:

```suggestion
    sess_time = ota and ota.timeslot.local_start_time()
```

Also, you'll need to handle the case where `ota is None` a few more times since it'll lead to `sess_time` being `None` in the code that follows.

Reply:
hmm - it would be very odd for ota to be None since this is used to report on a session that _happened_ already (and was set up well enough to happen in MeetEcho), but I'll look into better resiliency.

Reply:
It just struck me as odd that there's the guard in this line.